src/components/dashboard/DashBody.js

```diff
@@ -58,7 +60,7 @@ const BuildWidgets = widgetArray =>
       'Clock': <Clock { ...widget } key={ widget.title } />
     }[ widget.type ]))
 
-class DashboardBody extends Component {
+class DashBody extends Component {
```

Comment:
It was changed to be consistent with the other components which are named 'Dash*' instead of 'Dashboard*'